website/blog/2024-03-18-deploying-offline-with-docker/index.md

```diff
@@ -0,0 +1,69 @@
+---
+authors: [wwayne]
+tags: [deployment]
+---
+
+# Deploying offline with Docker
+
+<div align="center">
+
+![No internet access](./no-internet.png)
+
+</div>
+
+Are you facing a situation where your work environment lacks internet access, and you're wondering if you can still deploy Tabby? Fear not, because the answer is YES! 🐱📣
+
+## Prerequisite📋
+
+* Docker installed on both the internet-connected computer and the offline computer.
+
+## The Game Plan🚀
```

Comment:
Consider making 'The Game Plan' and 'Step-ty-step guideline' into one whole section? The three steps in 'The Game Plan' can be shown as a high-level overview, and the content below is more detailed steps.

Reply:
updated